kubernetes/services/node_manager.yaml

```diff
@@ -18,6 +18,8 @@ data:
     spring.servlet.multipart.file-size-threshold=2KB
     spring.servlet.multipart.max-file-size=200MB
     spring.servlet.multipart.max-request-size=215MB
+    microservices.dataplane.service.url=http://dataplanemanager-service.default.svc.cluster.local:9010/network-configuration
```

Comment:
Please check PM's yaml file. The port number should be adjusted to 300XX for K8s deployment.

Reply:
Fixed and updated PR.

Reply:
I think either 90xx or 300xx are both ok here, but 300xx will pass though API gw.

Reply:
My bad. 90xx works here.